docs/monitoring/initial-setup-configuration.ipynb

```diff
@@ -28,8 +28,10 @@
     "## Enabling model monitoring\n",
     "\n",
     "Model activities can be tracked into a real-time stream and time-series DB. The monitoring data\n",
-    "is used to create real-time dashboards and track model accuracy and drift. \n",
-    "To set the tracking stream options, specify the following function spec attributes:\n",
+    "is used to create real-time dashboards, track model accuracy, detect drift, and analyze performance. \n",
+    "\n",
+    "To monitor a deployed model, apply `set_tracking()` on your serving function and specify the function spec attributes:\n",
+    "\n",
     "    \n",
     "   `fn.set_tracking(stream_path, batch, sample)`\n",
```

Comment:
`tracking_policy` param is missing from the function signature 
```suggestion
    "   `fn.set_tracking(stream_path, batch, sample, tracking_policy)`\n",
```

Reply:
In addition, you can add a description for the `tracking_policy` param in the next lines (can't comment on unchanged lines in PR review):
**`tracking_policy`** - optional, model tracking configurations, such as setting the scheduling policy of the model monitoring batch job.